user-guide/modules/ROOT/pages/getting-started.adoc

```diff
@@ -0,0 +1,886 @@
+
+== Package Managers
+
+While we provide instructions for downloading and building the libraries from source, installing Boost using a package manager might simplify the process and ensure that all dependencies are installed correctly in some cases.
+In this guide, we will discuss two types of package managers: pass:[C++] package managers such as Conan and Vcpkg, and system package managers such as Homebrew and APT.
+
+While using a package manager can simplify the installation process, there are some potential downsides to consider, such as a lack of control over the installation process and potential version conflicts with other installed software.
+We will discuss these in more detail below.
+
+=== pass:[C++] Package Managers
+
+pass:[C++] package managers are a popular choice for managing dependencies in pass:[C++] projects.
+They provide a simple and consistent way to download, configure, and install libraries and their dependencies that's replicable on various platforms.
+However, these installation methods are not officially supported, which means the packages are not regularly tested by Boost authors.
+Another downside is the possibility of missing or outdated packages.
+
+==== Vcpkg
+
+To install Boost with Vcpkg, you can run a command like:
+
+[source]
+----
+vcpkg install boost
+----
+
+You can also install individual boost modules by providing their name as a suffix:
+
+[source]
+----
+vcpkg install boost-variant2
+----
+
+To verify access to Boost libraries, enter:
+
+[source]
+----
+vcpkg search boost
+----
+
+You should see a full list of the Boost libraries you have installed.
+
+A better approach, however, is adding these dependencies to your project manifest file.
+Follow the instruction in https://learn.microsoft.com/en-us/vcpkg/users/manifests[the introduction guide] to manifest files.
+
+==== Conan
+
+You can install Boost with Conan through the https://conan.io/center/boost[boost] package.
+
+=== System Package Managers
+
+Depending on your operating system, you may have different package managers available.
+For instance, on Debian-based Linux distributions, APT is a popular choice, while macOS users often use Homebrew, and Windows users may prefer Chocolatey.
+Depending on your version of Unix, you can use a package manager such as *apt*, *dnf*, *yum*, or *pacman*.
+While using a system package manager has its advantages, there are also some drawbacks to consider.
+These include potential version conflicts with system libraries, lack of control over the installation process, and the risk of using outdated or unsupported versions of the Boost libraries.
+
+==== Linux
+
+Update your package list to ensure you're getting the most recent version of the Boost libraries available in the repositories.
+
+[tabs]
+========
+Ubuntu::
++
+--
+To install the Boost pass:[C++] libraries onto Debian-based systems (such as Debian, Ubuntu), you can use the package manager apt.
+
+[source]
```

Comment:
I'd merge the `sudo apt update` and `sudo apt install libboost-all-dev` tabs in a single thing.